router/pkg/routerconfig/client.go

```diff
@@ -21,3 +21,43 @@ type ConfigNotFoundError interface {
 	error
 	FederatedGraphId() string
 }
+
+func GetDefaultConfig() []byte {
+	return []byte(`
```

Comment:
Can you define the config in code instead of JSON bytes?
Using JSON bytes can obfuscate breaking changes in the future as we wouldn't see if the bytes are incompatbile.